CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Added
 
-- Nothing.
+- [#65](https://github.com/zendframework/zend-expressive-zendviewrenderer/pull/65)
+  enables usage of non-interop psr containers with view helper plugin manager
```

Comment:
"non-interop psr containers" feels like it is enabling usage of non-standard containers. I'd rephrase to something like "usage of any PSR-11 container with the view helper plugin manager".